decidim-blogs/spec/types/integration_schema_spec.rb

```diff
@@ -150,6 +155,8 @@
           edges {
             node{
```

Comment:
Same here

Reply:
done in [7c7ad42](https://github.com/decidim/decidim/pull/14358/commits/7c7ad42d2220e4d982785f81246a9e7418c82d76)